boards/weact/mini_stm32h7b0/doc/index.rst

```diff
@@ -0,0 +1,241 @@
+.. _mini_stm32h7b0:
+
+WeAct Studio MiniSTM32H7B0 Core Board
+#####################################
+
+Overview
+********
+
+The MiniSTM32H7B0 Core board is a compact development board equipped with
+an STM32H7B0VBT6 microcontroller. It features a variety of peripherals,
+including a user LED and button, a display, and external SPI and QuadSPI
+NOR flash memory.
+
+Key Features
+
+- STM32 microcontroller in LQFP100 package
+- USB OTG or full-speed device
+- 1 user LED
+- User, boot, and reset push-buttons
+- 32.768 kHz and 25MHz HSE crystal oscillators
+- External NOR Flash memories: 64-Mbit Quad-SPI and 64-Mbit SPI
+- Board connectors:
+   - Camera (8 bit) connector
+   - ST7735 TFT-LCD 160 x 80 pixels (RGB565 3-SPI)
+   - microSD |trade| card
+   - USB Type-C Connector
+   - SWD header for external debugger
+   - 2x 40-pin GPIO connector
+
+.. figure:: img/stm32h7xx.webp
+      :align: center
+      :alt: MiniSTM32H7B0 Core Board
+
+      MiniSTM32H7B0 Core Board (Credit: WeAct Studio)
+
+More information about the board can be found on the `Mini_STM32H7B0 website`_.
+
+Hardware
+********
+
+The MiniSTM32H743 Core board provides the following hardware components:
+
+- STM32H7B0VBT6 in LQFP100 package
+- ARM 32-bit Cortex-M7 CPU with FPU
+- Chrom-ART Accelerator
+- Hardware JPEG Codec
+- 280 MHz max CPU frequency
+- VDD from 1.62 V to 3.6 V
+- 128 KB Flash
+- 1.4 MB SRAM
+- High-resolution timer (2.1 ns)
+- 32-bit timers(2)
+- 16-bit timers(10)
+- SPI(5)
+- I2C(4)
+- I2S (4)
+- USART(5)
+- UART(5)
+- USB OTG Full Speed and High Speed(1)
+- CAN FD(2)
+- SAI(4)
+- SPDIF_Rx(4)
+- HDMI_CEC(1)
+- Dual Mode Quad SPI(1)
+- Camera Interface
+- GPIO (up to 80) with external interrupt capability
+- 16-bit ADC(2) with 16 channels
+- 12-bit DAC with 2 channels(2)
+- True Random Number Generator (RNG)
+- 16-channel DMA
+- LCD-TFT Controller with XGA resolution
+
+More information about STM32H7BO can be found here:
+
+- `STM32H7B0VB on www.st.com`_
+- `STM32H7B0VB reference manual`_
+- `STM32H7B0VB datasheet`_
+
+Supported Features
+==================
+
+The mini_stm32h7b0 board configuration supports the following hardware features:
```

Comment:
*board target, and enclose board name with \`\`

Reply:
@nordicjm currently it is consistent with ever other ST and WeAct boards doc. I can change it if its required but it breaks consistency. What I will do atm is make it on par with a43009079f6670ef1085603b0474d147ffdb76f4

Reply:
board configuration does not exist and is invalid, board target is correct, see https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#board-terminology

Reply:
@nordicjm fixed